tests/main/views/test_drafts.py

```diff
@@ -1085,6 +1140,17 @@ def test_should_not_copy_draft_documents(self):
         assert "serviceDefinitionDocumentURL" not in data['services']
         assert "sfiaRateDocumentURL" not in data['services']
 
+    @mock.patch('app.db.session.commit')
+    def test_copy_draft_service_should_catch_db_integrity_error(self, db_commit):
+        db_commit.side_effect = IntegrityError("Could not commit", orig=None, params={})
+        res = self.client.post(
+            '/draft-services/%s/copy' % self.draft_id,
```

Comment:
Yeah that'd be nice.  There are a few places they get used.  We probably don't need to go through them all with a fine toothcomb, but if you spot one no harm in fixing up!